workers/unity/Packages/io.improbable.gdk.deploymentlauncher/DeploymentLauncherWindow.cs

```diff
@@ -882,6 +888,23 @@ private string GetStatusMessage()
             return sb.ToString();
         }
 
+        private void MarkConfigAsDirty()
+        {
+            EditorUtility.SetDirty(launcherConfig);
+            lastEditTime = DateTime.Now;
+        }
+
+        private void SaveChanges()
```

Comment:
```suggestion
        private void TrySaveChanges()
```

Nit - clearer naming, doesn't always save.